sponsorblockcast.sh

```diff
@@ -7,6 +7,18 @@ SBCSCANINTERVAL="${SBCSCANINTERVAL:-300}"
 SBCDIR="${SBCDIR:-/tmp/sponsorblockcast}"
 SBCCATEGORIES="${SBCCATEGORIES:-sponsor}"
 
+sysname=$(uname -s)
+if [ "$sysname" = "Darwin" ]; then
+  if which ggrep gsed > /dev/null; then
+    alias grep=ggrep
+    alias sed=gsed
+    shopt -s expand_aliases
```

Comment:
`shopt` isn't available for shells other than bash, but shouldn't be required since this script runs with `/bin/sh` making bash run in POSIX compatibility mode, always enabling aliases.

Reply:
Good point. Removed.